src/compute/src/render/continual_task.rs

```diff
@@ -445,8 +445,6 @@ impl<D: Ord> SinkState<D, Timestamp> {
     }
 
     /// Returns data to write to the output, if any, and the new upper to use.
-    ///
-    /// TODO(ct): Remove the Vec allocation here.
```

Comment:
I tried, it's tricky. Also probably not worth it given that this will start handing it off to the output of the operator so it can be written down by a multi-worker persist_sink and it will all have to be owned anyway.